parent/pom.xml

```diff
@@ -692,8 +701,8 @@
 						<artifactId>tycho-packaging-plugin</artifactId>
 						<version>${tychoVersion}</version>
 						<configuration>
-							<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm'-B${BUILD_NUMBER}'</format>
-							<archiveSite>true</archiveSite>
+							<jgit.dirtyWorkingTree>error</jgit.dirtyWorkingTree>
+							<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm'-ci'</format>
```

Comment:
Continuous Integration, lowercase. Because then it's of a "newer" value in the ASCII table than 'LOCAL' which guarantees anything built in Jenkins with the same time stamp as something you build locally will be seen as newer.